.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,103 @@
+{
+  "name": "SmartOrder Container",
+  "service": "smart-order",
+  "runServices": ["smart-order"],
+  "dockerComposeFile": "docker-compose.yml",
+  "workspaceFolder": "/workspaces/SmartOrder",
+  "onCreateCommand": "/bin/bash .devcontainer/bootstrap.sh",
+  "containerEnv": {
+    "VSCODE": "true"
+  },
+  "mounts": [
+    "source=${localWorkspaceFolder}/../,target=/workspaces,type=bind,consistency=cached"
+  ],
+  "customizations": {
+    "vscode": {
+      "settings": {
+        "terminal.integrated.shell.linux": "/bin/zsh"
+      },
+      "extensions": [
```

Comment:
Are these all the vs code plugins you're bundling in the container? I thought vs code was still running locally?

Reply:
No, not bundling. These are just settings that get loaded when you load the container.  This tells vscode to install these extensions (but only in the context of the container).

There's still a few in here I meant to remove, I just noticed...

Reply:
That's kinda what I meant. Inside the container, vs code is being bundled with all those extensions. I'll read more on Dev Containers but clarification on them is a big part of our call today.